policyengine_canada/variables/gov/provinces/mb/tax/income/credits/tuition_amount/mb_tuition_amount_credit.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_canada.model_api import *
+
+
+class mb_tuition_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Manitoba tuition amount credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.MB
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.mb.tax.income.credits.tuition_amount
+
+        tuition = person("tuition_expenses", period)
+        tuition_eligible = tuition > p.eligible_tuition_amount
+
+        # check if full-time student
+        full_time = person("is_full_time_student", period)
+        part_time = ~full_time
+
+        # check if disabled
+        disabled = person("is_disabled", period)
+        nondisabled = ~disabled
+
+        return tuition_eligible * (
+            tuition
```

Comment:
i would suggest using a select statement here instead
select([full_time, ~full_time & disabled, ~full_time & ~disabled ...
with the corresponding outputs